Documentation edits for concise, active voice (#355)
dashbitco/broadway: guides/examples/amazon-sqs.md

and [`Batcher concurrency`](https://hexdocs.pm/broadway/Broadway.html#module-batcher-concurrency)
for details.

> Here's an example on how you tune them according to

Reviewer: ```suggestion
Here's an example on how to tune them according to
```

Think this reads better?

In order to get a good set of configurations for your pipeline, it's
important to respect the limitations of the servers you're running,
as well as the limitations of the services you're providing/consuming

> data to/from. Measure your pipeline with [telemetry](https://hexdocs.pm/telemetry/readme.html) to ensure your changes are effective. (It comes standard.)

Reviewer: I really don't think the parentheses there work well. What about

```suggestion
data to/from. To measure your pipeline and ensure your changes are effective, use the [telemetry](https://hexdocs.pm/telemetry/readme.html) integration that comes with Broadway.
```
